.github/workflows/deploy-to-sandbox.yml

```diff
@@ -0,0 +1,170 @@
+name: Deploy to Sandbox Environment
+
+on:
+  push:
+    branches: [ main, develop, DEV-* ]
```

Comment:
Do we really want this to deploy immediately on pushes to `DEV-*` branches? Seems like that could cause unintentional race conditions between developers. 😝